src/lightning/fabric/fabric.py

```diff
@@ -363,6 +363,35 @@ def backward(self, tensor: Tensor, *args: Any, model: Optional[_FabricModule] =
 
         self._precision.backward(tensor, module, *args, **kwargs)
 
+    def clip_gradients(
```

Comment:
would be nice to have this covered in `tests/tests_fabric/test_fabric.py` with a strategy mock.